library/common/config/config.cc

```diff
@@ -136,6 +118,25 @@ R"(
 )";
 
 const char* config_template = R"(
+!ignore base_protocol_options_defs:
+    typed_extension_protocol_options: &base_protocol_options
```

Comment:
You can actually delete this key and either make it a sequence (`-`) or move the whole thing up a level. But not a big deal. Not even a nit really, just an FYI. :)